src/components/common/Button.tsx

```
@@ -1,6 +1,6 @@
 import {tv, type VariantProps} from 'tailwind-variants/lite';
 
-const button = tv({
+export const buttonStyles = tv({
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)